src/app.module.ts

```diff
@@ -40,7 +42,19 @@ import { OidcHelperModule } from './oidc/OidcHelper.module';
     SignModule,
     PermissionsModule,
     OidcHelperModule,
+    HttpModule
   ],
   providers: [DbLookupValidator, IsValidCodesValidator],
 })
-export class AppModule {}
+export class AppModule implements NestModule {
+  configure(consumer: MiddlewareConsumer) {
+    consumer.apply(MaintenanceMiddleware).exclude(
+      { path: "/auth-mgmt/authentication/login-state", method: RequestMethod.GET },
```

Comment:
If mock permissions are enabled, I think that endpoint will need to be here, too:
```suggestion
      { path: "/auth-mgmt/authentication/login-state", method: RequestMethod.GET },
      { path: "/auth-mgmt/permissions", method: RequestMethod.GET },
```